<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calc</class>
 <widget class="QWidget" name="Calc">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calc</string>
  </property>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_7">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>380</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="numDigits">
    <number>5</number>
   </property>
   <property name="digitCount">
    <number>5</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_9">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Member Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Member Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Member Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Member Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Member Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Member Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Member Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Member Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Member Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Member Cost</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Member Cost</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Member Cost</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Member Cost</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Member Cost</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Member Cost</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Member Cost</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>360</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Member Cost</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>410</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Member Cost</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>€</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>€</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>€</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>€</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>€</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>€</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>€</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>380</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>€</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>430</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>€</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>41</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="text">
    <string>€</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="displayFormat">
    <string>HH:mm:ss</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>240</y>
     <width>131</width>
     <height>111</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>checkBox</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>spinBox_2</tabstop>
  <tabstop>checkBox_3</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>spinBox_3</tabstop>
  <tabstop>checkBox_4</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>spinBox_4</tabstop>
  <tabstop>checkBox_5</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>spinBox_5</tabstop>
  <tabstop>checkBox_6</tabstop>
  <tabstop>lineEdit_6</tabstop>
  <tabstop>spinBox_6</tabstop>
  <tabstop>checkBox_7</tabstop>
  <tabstop>lineEdit_7</tabstop>
  <tabstop>spinBox_7</tabstop>
  <tabstop>checkBox_8</tabstop>
  <tabstop>lineEdit_8</tabstop>
  <tabstop>spinBox_8</tabstop>
  <tabstop>checkBox_9</tabstop>
  <tabstop>lineEdit_9</tabstop>
  <tabstop>spinBox_9</tabstop>
  <tabstop>timeEdit</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
